docs/learn-treasury.md

```diff
@@ -114,7 +114,9 @@ After being turned into a council motion, the proposal needs 51% of the council
 pass, or 51% to reject it to fail. If majority is not reached, the proposal remains in limbo -
 neither accepted nor rejected until more council members weigh in.
 
-![Motion in action](/img/treasury/motion.jpg)
+![Council](/img/treasury/new-council.jpg)
```

Comment:
Same point here - the image doesn't exactly fit with the text that's describing it